gis/openstreetmap-carto/Portfile

```diff
@@ -0,0 +1,203 @@
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/tourism/*] \
+        ${destroot}${prefix}/share/${name}/symbols/tourism
+
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/leisure
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/leisure/*] \
+        ${destroot}${prefix}/share/${name}/symbols/leisure
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/historic
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/historic/*] \
+        ${destroot}${prefix}/share/${name}/symbols/historic
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/natural
+        xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/natural/*] \
+        ${destroot}${prefix}/share/${name}/symbols/natural
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/office
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/office/*] \
+        ${destroot}${prefix}/share/${name}/symbols/office
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/highway
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/highway/*] \
+        ${destroot}${prefix}/share/${name}/symbols/highway
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/shields
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/shields/*] \
+        ${destroot}${prefix}/share/${name}/symbols/shields
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/man_made
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/man_made/*] \
+        ${destroot}${prefix}/share/${name}/symbols/man_made
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/shop
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/shop/*] \
+        ${destroot}${prefix}/share/${name}/symbols/shop
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/amenity
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/amenity/*] \
+        ${destroot}${prefix}/share/${name}/symbols/amenity
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/barrier
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/barrier/*] \
+        ${destroot}${prefix}/share/${name}/symbols/barrier
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/religion
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/religion/*] \
+        ${destroot}${prefix}/share/${name}/symbols/religion
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/generating_patterns
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/generating_patterns/*] \
+        ${destroot}${prefix}/share/${name}/symbols/generating_patterns
+    xinstall -d ${destroot}${prefix}/share/${name}/symbols/place
+    xinstall -m 0644 \
+        {*}[glob ${worksrcpath}/symbols/place/*] \
+        ${destroot}${prefix}/share/${name}/symbols/place
+    file rename ${destroot}${prefix}/share/${name}/external-data.yml \
+        ${destroot}${configdir}/external-data.yml.dist
+}
```

Comment:
this `destroot` phase is rather involved and not very readable... If I see this correctly, you pretty much install the whole caboodle that comes in the archive. 

So can you not just replace this with (please not this is untested, but you'll get the point...):
```
xinstall -d ${destroot}${prefix}/etc/${name}
file copy ${worksrcpath} ${destroot}${prefix}/share/${name}
```

if there are really files that you don't need then you can just remove these in the `post-destroot` phase (you can also change permissions of the python scripts there if needed). Anyway, it would be good if you could take another look at this and try to simplify this a bit.

Reply:
Thanks.  The configuration is a lot easier to follow now :D